gateway-modules/apim-ballerina-configs/configs/org/wso2/carbon/apimgt/gateway/microgateway/microUtil.bal

```diff
@@ -0,0 +1,178 @@
+package org.wso2.carbon.apimgt.gateway.microgateway;
+import ballerina.lang.errors;
+import ballerina.lang.system;
+import ballerina.lang.strings;
+import ballerina.lang.jsons;
+import org.wso2.carbon.apimgt.gateway.dto as dto;
+import org.wso2.carbon.apimgt.gateway.holders as holders;
+import ballerina.lang.files;
+import ballerina.lang.blobs;
+import org.wso2.carbon.apimgt.ballerina.util;
+import org.wso2.carbon.apimgt.gateway.utils as gatewayUtil;
+import org.wso2.carbon.apimgt.gateway.constants as Constants;
+errors:TypeCastError err;
+
+function loadConfigs () {
+    string name = system:getEnv(Constants:GW_HOME);
+    files:File t = {path:name + "/microgateway/microConf.json"};
+
+    if (files:exists(t)) {
+        try {
+            files:open(t, "r");
+            var content, n = files:read(t, 100000000);
+
+            string strConf = blobs:toString(content, "utf-8");
+            json conf = util:parse(strConf);
+            dto:GatewayConfDTO gatewayConfDTO = gatewayUtil:fromJsonToGatewayConfDTO(conf);
+            holders:setGatewayConf(gatewayConfDTO);
+        } catch (errors:Error error) {
+            system:println("WARNING : analytics configuration not found");
+        }
+    }
+}
+
+function isSwaggerJsonFile (json apiData) (boolean) {
+    try {
+        string swagger = jsons:getString(apiData,"$.swagger");
+    } catch (errors:Error error) {
+        return false;
+    }
+    return true;
+}
+
+function readFromJSONFile (string filePath) (json) {
+    string gwHome = system:getEnv(Constants:GW_HOME);
+    try {
+        files:File jsonFile = {path:gwHome + "/microgateway/" + filePath};
+        files:open(jsonFile, "r");
+
+        var content, n = files:read(jsonFile, 100000000);
+        string strAPIData = blobs:toString(content, "utf-8");
+        json apiData = util:parse(strAPIData);
+
```

Comment:
Shall we remove the additional line here as well?

Reply:
okay